### PR TITLE
Handle single unlisted images in geom_phylopic

### LINKED ISSUE #75: geom_phylopic requires image to be in a list
Right now, if you try to use `geom_phylopic()` with a single image, you need to wrap that image in a `list()`. We should resolve this by doing the wrapping ourselves if necessary if an individual Picture or array image is passed to the function.

### PR DESCRIPTION
**Pull request by willgearty**
This wraps the image in a list if the user doesn't do it beforehand.

Fixes #75.

### CODE REVIEW

File: R/geom_phylopic.R [hunk 1]
```diff
@@ -68,6 +68,11 @@ geom_phylopic <- function(mapping = NULL, data = NULL,
   if (!is.logical(remove_background)) {
     stop("`remove_background` should be a logical value.")
   }
+  # We need to do this before passing to the layer
+  dots <- list(...)
+  if ("img" %in% names(dots) && (is(dots$img, 'Picture') || is.array(dots$img))) {
```
Comment (LewisAJones) [inline R/geom_phylopic.R:73]: Is `&& (is(dots$img, 'Picture') || is.array(dots$img))` strictly necessary here? Shouldn't this be handled as an error elsewhere? If not, shouldn't we have error handling here? `img` should conform to one of these?

File: R/geom_phylopic.R [hunk 2]
```diff
@@ -76,10 +81,10 @@ geom_phylopic <- function(mapping = NULL, data = NULL,
     position = position,
     show.legend = show.legend,
     inherit.aes = inherit.aes,
-    params = list(
+    params = c(
       na.rm = na.rm,
       remove_background = remove_background,
-      ...
+      dots
```
Comment (LewisAJones) [inline R/geom_phylopic.R:87]: should we make this `args = dots`? Or similar?

**Review by LewisAJones — CHANGES_REQUESTED** [review]
Hi @willgearty, looks good, just some minor comments!

File: R/geom_phylopic.R (hunk 1 above)
Comment (willgearty) [reply R/geom_phylopic.R:80]: You're right, this checking is already handled in GeomPhylopic, so I've changed this to just check if it is a list already.

File: R/geom_phylopic.R (hunk 2 above)
Comment (willgearty) [reply R/geom_phylopic.R:95]: dots is already a named list, so here we are just concatenating it with the other named items.

Comment (LewisAJones) [reply R/geom_phylopic.R:102]: Yes, it was just more for clarity, but it is also fine!

**Review by LewisAJones — APPROVED** [review]
Looks good @willgearty! Thanks for updating the vignette here too!

**Pull request merged**